src/data/finset/sort.lean

```diff
@@ -175,6 +177,21 @@ begin
     exact (ne_of_lt (mono_of_fin_strict_mono s h ji) hj).elim }
 end
 
+/-- Any increasing map between `fin k` and a finset of cardinality `k` has to coincide with
+the increasing bijection `mono_of_fin s h`. -/
+lemma mono_of_fin_unique' [decidable_linear_order α] {s : finset α} {k : ℕ} (h : s.card = k)
+  {f : fin k → α} (fmap : set.maps_to f set.univ ↑s) (hmono : strict_mono f) :
+  f = s.mono_of_fin h :=
```

Comment:
This lemma was previously in `data.fintype.basic` for no good reason, except that it used `fintype.univ` to obtain a `finset (fin k)`. I've provided another way to get that, and moved this back to where it belongs.